src/installer.js

```diff
@@ -276,11 +277,12 @@ var createBinary = function (options, dir, callback) {
   var binDest = path.join(binDir, options.name)
   options.logger('Symlinking binary from ' + binSrc + ' to ' + binDest)
 
-  async.waterfall([
-    async.apply(fs.ensureDir, binDir),
-    async.apply(fs.symlink, binSrc, binDest, 'file')
-  ], function (err) {
-    callback(err && new Error('Error creating binary file: ' + (err.message || err)))
+  mkdirp(binDir, '0755', function (err, made) {
```

Comment:
Ah, but `fs-extra` doesn't have chmod options. Oh well, another dependency it is then.

Reply:
Yes, we're going to need a new dependency until `fs-extra` decides to include chmod options to their commands